docs/src/pages/components/accordion/accordion.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Accordion React component
+components: Accordion, AccordionActions, AccordionDetails, AccordionSummary
+---
+
+# Accordion
+
+<p class="description">Accordions contain creation flows and allow lightweight editing of an element.</p>
+
+[An accordion](https://material.io/archive/guidelines/components/expansion-panels.html) is a lightweight container that may either stand alone or be connected to a larger surface, such as a card.
+
+> **Note:** Accordions are no longer documented in the [Material Design guidelines](https://material.io/), but Material-UI will continue to support them.
```

Comment:
I think that we should add that it was formerly known as the expansion panel to help people upgrade.

```suggestion
> **Note:** Accordions are no longer documented in the [Material Design guidelines](https://material.io/), but Material-UI will continue to support them. It was formerly known as the "expansion panel". 
```